cmake/modules/packages/FindBRUNSLI.cmake

```
@@ -0,0 +1,45 @@
+set(BRUNSLI_NAME BRUNSLI)
+
+find_path(BRUNSLI_INCLUDE_DIR brunsli/encode.h
+    HINTS ${BRUNSLI_ROOT}
+    PATH_SUFFIXES ${BRUNSLI_NAME}/include include
+)
+
+find_library(BRUNSLI_ENC_LIB 
+    NAMES brunslienc-c
+    HINTS ${BRUNSLI_ROOT}
+    PATH_SUFFIXES ${BRUNSLI_NAME}/lib lib
+)
+
+find_library(BRUNSLI_DEC_LIB 
+    NAMES brunslidec-c
+    HINTS ${BRUNSLI_ROOT}
+    PATH_SUFFIXES ${BRUNSLI_NAME}/lib lib
+)
+
+set(BRUNSLI_LIBRARIES ${BRUNSLI_ENC_LIB} ${BRUNSLI_DEC_LIB})
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(BRUNSLI
+    REQUIRED_VARS   BRUNSLI_LIBRARIES BRUNSLI_INCLUDE_DIR
+)
+mark_as_advanced(BRUNSLI_INCLUDE_DIR BRUNSLI_LIBRARIES)
```

I believe this should avoid the expension of BRUNSLI_ENC_LIB and BRUNSLI_DEC_LIB in BRUNSLI_LIBRARIES, because if one would be missing, find_package_handle_standard_args() wouldn't realize it. 
```suggestion

include(FindPackageHandleStandardArgs)
find_package_handle_standard_args(BRUNSLI
    REQUIRED_VARS BRUNSLI_ENC_LIB BRUNSLI_DEC_LIB BRUNSLI_INCLUDE_DIR
)
mark_as_advanced(BRUNSLI_INCLUDE_DIR BRUNSLI_ENC_LIB BRUNSLI_DEC_LIB)
```